### DISCUSSION TITLE
MT7610U in KALI LINUX canno recognize

### DISCUSSION THREAD

**Discussion by ghost** [ZerBea/hcxdumptool#409] [General]
Physical machine run Kali 2023.4,  use wireless card: MT7610U,  Insertion canno recognize 
I remember clearly two days ago it can be recognized
But today it canno be recognized

When inserting USB, the wireless card light not light up
Driver issue or wireless card  firmware issue ?
I remember clearly two days ago OK !
I failed today

But In Windows10, this wireless card normally working

**Comment by ghost** [comment]
More eerie is that
Use Kali2023.4
$ uname -a
Linux kali 6.5.0-kali1-amd64

Use MT7610U, test in VirtualBox and VMware,  it can recognized MT7610U
But by physical machine run Kali2023.4 cannot recognize MT7610U !
rt5572 can recognize

**Reply by ZerBea** [reply]
Two possible causes:
KALI -> not customized
VM -> not a good idea to access hardware via a VM.

Running Arch Linux and Raspbian, everything is fine:
https://github.com/ZerBea/hcxdumptool/discussions/361#discussioncomment-7550957
https://github.com/ZerBea/hcxdumptool/discussions/361#discussioncomment-7551660

**Comment by ghost** [comment]
This USB interface type wireless card design seems  have some flaws, and kernel driver not been updated to solve this issues !

**Comment by ZerBea** [comment]
Linux kali 6.5.0-kali1-amd64 is outdated and it does not receive updates any longer.
This are the supported Linux kernels which receive updates (that include driver fixes):
https://www.kernel.org/
Your kernel 6.5.0 is not among them!

**Comment by ghost** [comment]
Thanks, I may need some time test it
